package.json

```diff
@@ -27,7 +27,14 @@
     "require": [
       "babel-register",
       "babel-polyfill"
-    ]
+    ],
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-3",
+        "react"
+      ]
+    }
```

Comment:
sorry first timer here, had some difficulties that's why it took this long... should be OK now tho! :)